src/OptFrame/Heuristics/SA/BasicSimulatedAnnealing.hpp

```diff
@@ -65,7 +74,7 @@ class BasicSimulatedAnnealing: public SingleObjSearch<R, ADS, S>
 	{
 	}
 
-	pair<S, Evaluation>* search(SOSC& stopCriteria, const S* _s = nullptr,  const Evaluation* _e = nullptr)
+	pair<S, Evaluation>* git search(SOSC& stopCriteria, const S* _s = nullptr,  const Evaluation* _e = nullptr)
```

Comment:
git??

Reply:
How have I not seen this?
Sorry, I'm changing it to WIP until I fix it, I guess I may have forgotten to push something.